### ISSUE TITLE
Legacy warning

### ISSUE THREAD

**Issue by omidkrad** [SuperPaintman/winston-seq#2]
I get the following message when trying to use winston-seq:

> Seq is a legacy winston transport. Consider upgrading:
> Upgrade docs: https://github.com/winstonjs/winston/blob/master/UPGRADE-3.0.md

**Comment by Apollon77** [comment]
Yes, that would be awesome @SuperPaintman do you think you can do an update here?

**Comment by liammclennan** [comment]
https://github.com/datalust/winston-seq supports winston 3 and is published on npm as @datalust/winston-seq. Please try it out and send through your feedback.
